mobile/library/common/config/BUILD

```diff
@@ -6,25 +6,15 @@ envoy_package()
 
 envoy_cc_library(
     name = "config_lib",
```

Comment:
Is this basically now only used to store the `certificates.inc` file? If so, maybe it could be renamed to something like `certificates_lib`?